privacyidea/lib/tokens/motptoken.py

```diff
@@ -168,7 +179,11 @@ def update(self, param, reset_failcount=True):
         :return: nothing
         """
         if self.hKeyRequired is True:
```

Comment:
This condition is actually always true since `hKeyRequired` is set to `True` in the constructor.